lib/remote_files/file.rb

```diff
@@ -79,6 +81,8 @@ def retrieve!
           next unless file
           @content      = file.content
           @content_type = file.content_type
+          # :populate_stored_in is a boolean
+          @stored_in = file.stored_in if options.key?(:populate_stored_in) && (@options[:populate_stored_in])
```

Comment:
Not sure if this changed or if I just noticed but why do we access `@options[:populate_stored_in]` both here and [here](https://github.com/zendesk/remote_files/pull/26/files#diff-eb648593d3efb23402e60618788bbf355b778db93d29db25762761ec5c743c4fR17)? Do we not need the instance var or is it better to just check `key?` there as well and then store it to use here?

Reply:
That if condition was written before I implemented the instance var, I forgot to update that if condition after those changes.